awscli/examples/application-signals/batch-get-service-level-objective-budget-report.rst

```diff
@@ -0,0 +1,99 @@
+                                    "Dimensions": [{
+                                        "Name": "InstanceId",
+                                        "Value": "i-0e098765432522"
+                                    }]
+                                },
+                                "Period": 60,
+                                "Stat": "Average"
+                            },
+                            "ReturnData": true
+                        }]
+                    },
+                    "MetricThreshold": 200.0,
+                    "ComparisonOperator": "LessThanOrEqualTo"
+                },
+                "Goal": {
+                    "Interval": {
+                        "RollingInterval": {
+                            "DurationUnit": "DAY",
+                            "Duration": 7
+                        }
+                    },
+                    "AttainmentGoal": 99.0,
+                    "WarningThreshold": 50.0
+                }
+            },
+            {
+                "Arn": "arn:aws:application-signals:us-east-1:123456789101:slo/SLOName2",
+                "Name": "test",
+                "EvaluationType": "PeriodBased",
+                "BudgetStatus": "BREACHED",
+                "Attainment": 97.39583275,
+                "TotalBudgetSeconds": 86,
+                "BudgetSecondsRemaining": -2154,
+                "Sli": {
+                    "SliMetric": {
+                        "MetricDataQueries": [{
+                            "Id": "cwMetric",
+                            "MetricStat": {
+                                "Metric": {
+                                    "Namespace": "AWS/EC2",
+                                    "MetricName": "CPUUtilization",
+                                    "Dimensions": [{
+                                        "Name": "InstanceId",
+                                        "Value": "i-0e12345678922"
+                                    }]
+                                },
+                                "Period": 300,
+                                "Stat": "Average"
+                            },
+                            "ReturnData": true
+                        }]
+                    },
+                    "MetricThreshold": 5.0,
+                    "ComparisonOperator": "GreaterThan"
+                },
+                "Goal": {
+                    "Interval": {
+                        "RollingInterval": {
+                            "DurationUnit": "DAY",
+                            "Duration": 1
+                        }
+                    },
+                    "AttainmentGoal": 99.9,
+                    "WarningThreshold": 30.0
+                }
+            }
+        ],
+        "Errors": []
+    }
+
+For more information, see `<https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Application-Monitoring-Sections.html>`__ in the *Amazon CloudWatch User Guide*.
```

Comment:
```suggestion
For more information, see `Application Signals <https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Application-Monitoring-Sections.html>`__ in the *Amazon CloudWatch User Guide*.
```

All your for more information links are missing the title portion of the formatting and are therefore display broken. Please fix the URLs in all your submitted files to the following format: 

```
`This is the topic title <https://link.to.the/topic/page>`__
```
https://aws.github.io/aws-cli/docs_styleguide.html#for-more-information-link